okta/framework/ApiClient.py

```diff
@@ -9,8 +9,16 @@
 class ApiClient(object):
 
     def __init__(self, *args, **kwargs):
-        self.base_url = kwargs['base_url'] or args[0]
-        self.api_token = kwargs['api_token'] or args[1]
+        if 'pathname' not in kwargs:
+            raise ValueError('Invalid url')
```

Comment:
Let's make this more specific - `A pathname must be provided to create an ApiClient`